src/components/spaces/cell-zoom/OrganelleWrapper.tsx

```diff
@@ -341,28 +357,28 @@ export class OrganelleWrapper extends React.Component<OrganelleWrapperProps, Org
     // });
   }
 
-  // private getOrganelleFromMouseEvent(evt: any) {
-  //   const possibleTargets: OrganelleType[] = Object.keys(OrganelleType)
-  //     .map(key => OrganelleType[key])
-  //     .filter(organelle => this.organelleSelectorInfo[organelle])
-  //     .filter(organelle => {
-  //       const visibleModes = this.organelleSelectorInfo[organelle].visibleModes;
-  //       return !visibleModes || visibleModes.indexOf(rootStore.mode) > -1;
-  //     });
-  //   return possibleTargets.find((t) => {
-  //     return evt.target._organelle.matches({selector: this.organelleSelectorInfo[t].selector});
-  //   });
-  // }
+  private getOrganelleFromMouseEvent(evt: any): OrganelleType | undefined {
+    const {cellZoom} = this.stores;
+    const possibleTargets: OrganelleType[] = (Object.keys(this.organelleSelectorInfo) as OrganelleType[])
```

Comment:
This looks like another possible place to use the new enum pattern I suggested on Slack